src/actions/actions.ts

```diff
@@ -1247,7 +1248,10 @@ class MCommandMoveHalfPageDown extends BaseMovement {
   keys = ["ctrl+d"];
 
   public async execAction(position: Position, vimState: VimState): Promise<Position> {
-    return new Position(Math.min(TextEditor.getLineCount() - 1, position.line + vimState.settings.scroll), position.character);
+    return new Position(
+      Math.min(TextEditor.getLineCount() - 1, position.line + <number> Configuration.getInstance().get("scroll")),
```

Comment:
I think the number cast can go inline like so

`.get<number>("scroll")`

Or something like that???